stock_inventory_require_lot/models/stock_change_product_qty.py

```diff
@@ -0,0 +1,10 @@
+from odoo import models, fields
+
+
+class StockChangeProductQty(models.TransientModel):
+
+    _inherit = 'stock.change.product.qty'
+
+    tracking = fields.Selection(
+        related='product_id.tracking', readonly=True
```

Comment:
Valen, here it would be much better this way:

> tracking = fields.Selection(
related='product_id.tracking',
> readonly=True,
>)